apiv2/tests/helpers.py

```diff
@@ -0,0 +1,35 @@
+import operator
+
+
+# Helper for checking deep equality of python dict/list objects
+# Source: https://gist.github.com/samuraisam/901117/521ed1ff8937cb43d7fcdbc1a6f6d0ed2c723bae
+def deep_eq(_v1, _v2):
```

Comment:
Writing piecemeal tests for graphql response structures was getting tedious. I added this so we can validate literal responses.